src/parquet/CMakeLists.txt

```diff
@@ -25,3 +25,4 @@ install(FILES
   DESTINATION include/parquet)
 
 ADD_PARQUET_TEST(reader-test)
+target_link_libraries(reader-test ${LINK_LIBS} -lz)
```

Comment:
To be clear, you should be linking the external library target declared as part of the build toolchain setup I described below